### PR TITLE
REMO-11: Implement section accordion

### PR DESCRIPTION
**Pull request by chalalala**
## Changelog
- Created `EditableAccordion` component
- Created `SectionList` component with mock data

## Screenshot
**Expanded:**
image

**Collapsed:**
image

**Editing:**
image

### CODE REVIEW

File: components/EditableAccordion/EditableAccordion.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+              autoFocus
+            />
+          ) : (
+            <span
+              className="mb-px h-full w-full truncate"
+              onDoubleClick={onDoubleClickText}
+            >
+              {title}
+            </span>
+          )}
```
Comment (thuutri2710) [inline components/EditableAccordion/EditableAccordion.tsx:83]: maybe it's not implemented yet. I see u haven't updated `title` after changing it. Just a note to do it later

**Review by thuutri2710 — APPROVED** [review]
LGTM

File: components/EditableAccordion/EditableAccordion.tsx (hunk 1 above)
Comment (chalalala) [reply components/EditableAccordion/EditableAccordion.tsx]: Actually `title` is updated with `onChangeTitle` which will be triggered on input blur. But yeah, maybe it's better to use the internal state `editingTitle` for this to make sure the change is reflected correctly. Updated in [9235ae0](https://github.com/chalalala/remo/pull/7/commits/9235ae022989fe43be843ec0f6b276a8bc4486e2).

**Comment by chalalala** [comment]
@thuutri2710 I've just added the code to allow the user to submit changes in titles by pressing enter in this commit
[5d4f6d4](https://github.com/chalalala/remo/pull/7/commits/5d4f6d4b515c05e63c0f6ab2d632bdfa89dff83f). Please help me check it 🙇‍♀️

**Review by thuutri2710 — APPROVED** [review]
LGTM

**Pull request merged**